mac/build.sh

```diff
@@ -12,80 +12,44 @@ KEYMAN_MAC_BASE_PATH="$KEYMAN_ROOT/mac"
 . "$KEYMAN_ROOT/resources/build/build-help.inc.sh"
 . "$KEYMAN_ROOT/mac/mac-utils.inc.sh"
 
-# This script runs from its own folder
-cd "$(dirname "$THIS_SCRIPT")"
-
+builder_describe "Builds Keyman for macOS." \
+  "@/core:mac" \
+  "clean" \
+  "configure" \
+  "build" \
+  "publish   Publishes debug info to Sentry and builds DMG, .download_info for distribution" \
```

Comment:
Formerly `-upload-sentry` + `-deploy preprelease`.  As with #11421, these make sense paired as part of a similar, release-oriented action.